### PR TITLE
[full-ci] [tests-only] Used kebab case for example table header name

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
kebab-case is used for the example table header name.

Covered Suites:
- apiSpaces
- apiSpacesDavOperation
- apiSpacesShares

## Related Issue

- Part of https://github.com/owncloud/QA/issues/829

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpaces/filePreviews.feature [hunk 1]
```diff
@@ -16,11 +16,11 @@ Feature: Preview file in project space
...
 
   Scenario Outline: user can preview created txt files in the project space
-    Given user "Alice" has uploaded a file inside space "previews of the files" with content "test" to "<entity>"
-    When user "Alice" downloads the preview of "<entity>" of the space "previews of the files" with width "<width>" and height "<height>" using the WebDAV API
+    Given user "Alice" has uploaded a file inside space "previews of the files" with content "test" to "<file-name>"
+    When user "Alice" downloads the preview of "<file-name>" of the space "previews of the files" with width "<width>" and height "<height>" using the WebDAV API
     Then the HTTP status code should be "200"
     Examples:
-      | entity                | width | height |
+      | file-name             | width | height |
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpaces/filePreviews.feature:23]: we can have `filename` or only `file`
```suggestion
      | filename             | width | height |
```

Comment (grgprarup) [reply tests/acceptance/features/apiSpaces/filePreviews.feature:23]: Then I have to refactor again in merged PR.

Comment (grgprarup) [reply tests/acceptance/features/apiSpaces/filePreviews.feature:23]: I prefer to use `file-name`. There are more places where `file-name` is used.

Comment (saw-jan) [reply tests/acceptance/features/apiSpaces/filePreviews.feature:23]: either one is fine with me

**Review by saw-jan — APPROVED** [review]

**Pull request merged**